CMakeLists.txt

```diff
@@ -5,15 +5,21 @@ include("cmake/common.cmake")
 set(Cnl_DESTDIR ${CMAKE_BINARY_DIR}/install)
 file(MAKE_DIRECTORY ${Cnl_DESTDIR})
 
+message("CXX COMPILER = ${CMAKE_CXX_COMPILER}")
+message("C COMPILER = ${CMAKE_C_COMPILER}")
+message("INT128 = ${INT128}")
+message("STD = ${STD}")
+message("EXCEPTIONS = ${EXCEPTIONS}")
+
 ExternalProject_Add(
         Cnl
         PREFIX ${CMAKE_BINARY_DIR}/Cnl
-        GIT_REPOSITORY "https://github.com/johnmcfarlane/cnl.git"
+        GIT_REPOSITORY "https://github.com/hbe72/cnl.git"
         GIT_TAG "develop"
```

Comment:
Not sure if I've commented on this yet but you'll notice that where I pull in an external project in CNL ([example](https://github.com/johnmcfarlane/cnl/blob/develop/src/test/CMakeLists.txt#L81)), I specify the SHA explicitly. Here, that would mean changing `GIT_TAG` from `develop` to something like [`d5ae23f62075c74cd453723819ff6e2174ff83cb`](https://github.com/johnmcfarlane/cnl/tree/d5ae23f62075c74cd453723819ff6e2174ff83cb). I'm not at all sure that this is better but two benefits I've found are:

1. The code is more stable because if I push a change to CNL *develop* that breaks CDSP, it won't be pulled in automatically until you update your `GIT_TAG` entry manually.
2. It isn't necessary to fetch the dependency every time you build. (I'm not completely clear on the details of when the fetch occurs and how: it's either part of the CMake logic or disabled by adding the `UPDATE_COMMAND ""` line below but intuitively, you know from a fixed SHA that you have a fixed state whereas *develop* is a 'moving target'.)